ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml

```diff
@@ -0,0 +1,20 @@
+chain:
+  as: cucushift-installer-rehearse-nutanix-ipi-proxy-provision
+  steps:
+  - chain: ipi-conf-nutanix
+  - ref: ipi-install-rbac
+  - chain: nutanix-provision-bastionhost
+  - ref: mirror-images-by-oc-adm-in-bastion
```

Comment:
cluster install behind proxy also need the mirror operation? I think no.

Reply:
Good catch, I missed remove it